sklearn/datasets/tests/test_20news.py

```diff
@@ -88,3 +90,16 @@ def test_20news_normalization(fetch_20newsgroups_vectorized_fxt):
 
     assert_allclose_dense_sparse(X_norm, normalize(X))
     assert np.allclose(np.linalg.norm(X_norm.todense(), axis=1), 1)
+
+
+def test_20news_asframe(fetch_20newsgroups_vectorized_fxt):
```

Comment:
should we somehow check the column names? Do we test for column names in other places where we've been returning a data frame?

Reply:
There are some cases where it's tested, and there are others where it's not. I assume you suggest to check a subset of the column names, not all the 130k of them, right?